Lesson7/main.js

```diff
@@ -0,0 +1,34 @@
+const ul = document.getElementById("js-ul");
+const loading = document.getElementById("js-loading")
+const fragment = new DocumentFragment();
+const features = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+{to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
```

Comment:
IMO
After each process is made into a function, 
it is recommended to make local variables for variables that don't need to be global variables. 
By reducing the scope, you can reduce the occurrence of bugs.

Reply:
I tried to put some variable into local level, but it could be better
Please check my latest commitment b8d018d8d60ded9a6b6c019ca2f061d1184fbeca